actors/miner/src/lib.rs

```diff
@@ -883,32 +883,16 @@ impl Actor {
         Self::prove_replica_updates_inner(rt, updates)
     }
 
-    fn prove_replica_updates2<RT>(
-        rt: &RT,
-        params: ProveReplicaUpdatesParams2,
-    ) -> Result<BitField, ActorError>
-    where
-        // + Clone because we messed up and need to keep a copy around between transactions.
-        // https://github.com/filecoin-project/builtin-actors/issues/133
-        RT::Blockstore: Blockstore + Clone,
-        RT: Runtime,
-    {
-        let updates = params
-            .updates
-            .into_iter()
-            .map(|ru| ReplicaUpdateInner {
-                sector_number: ru.sector_number,
-                deadline: ru.deadline,
-                partition: ru.partition,
-                new_sealed_cid: ru.new_sealed_cid,
-                new_unsealed_cid: Some(ru.new_unsealed_cid),
-                deals: ru.deals,
-                update_proof_type: ru.update_proof_type,
-                replica_proof: ru.replica_proof,
-            })
-            .collect();
-        Self::prove_replica_updates_inner(rt, updates)
+    pub fn prove_replica_updates_deprecated(
```

Comment:
I did it this way for expedience, might be better to follow the patter in the account actor implementing a catch all method for all unhandled enums.